python/GafferSceneUI/VisualiserToolUI.py

```diff
@@ -85,6 +87,34 @@
 			"toolbarLayout:section", "Bottom",
 			"toolbarLayout:width", 100,
 
+		],
+		"mode" : [
+
+			"description",
+			"""
+			The method for displaying the data.
+
+			Float and integer data are displayed as grayscale. V2f data is displayed with
+			the `x` value for red, `y` value for green and `0` for blue. Vector data is
+			displayed with the `x` value for red, `y` value for green and `z` value for blue.
+			Color data is displayed directly.
+
+			- Auto : Same as `Color (Type Range)`.
+			- Color (Type Range) : Float, integer, V2f and color data is displayed without
+			modification. Vector data is remapped from `[-1, 1]` to `[0, 1]`.
+			- Color (Manual Range) : Values are remapped from the range `[valueMin, valueMax]` to
+			`[0, 1]`.
+			""",
+
+			"preset:Auto", GafferSceneUI.VisualiserTool.Mode.Auto,
+			"preset:Color (Type Range)", GafferSceneUI.VisualiserTool.Mode.ColorTypeRange,
+			"preset:Color (Manual Range)", GafferSceneUI.VisualiserTool.Mode.ColorManualRange,
```

Comment:
I know you're trying to convey the underlying mechanism in the name here, but I do wonder if maybe "Type Range" is a bit unobvious? I think maybe "Auto Range" conveys more quickly that the range will be picked for you, and then the tooltip can tell you the details of how.

Maybe just "Color" and "Color (Auto Range)" for the names?

I'm bikeshedding here, and others may disagree, so feel free to canvas other opinions.

Reply:
That makes sense to me, "Type Range" would probably be overly specific for most users. Changed in https://github.com/GafferHQ/gaffer/pull/6212/commits/79cec4c092a546c71f1fe031a903403d334e8ceb

Reply:
Could we put "Color (Auto Range)" after "Color" please? To me it is a variant of the first, so makes sense to come second.

Reply:
Right, yes, fixed in https://github.com/GafferHQ/gaffer/pull/6212/commits/a97c8f110a9ecc88f481a74dbaaef36cfaa97394